docs/pages/admin-guides/access-controls/access-request-plugins/datadog-hosted.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Access Requests with Datadog Incident Management (Hosted)
```

Comment:
I think the "Hosted" vs "Self-hosted" working can be improved. It's not clear for users what a hosted plugin is. Also self hosted is technically hosted by yourself, so they are not opposed concepts.

We would benefit for picking clear and well-defined terms, especially as one of our proposed Q4 goals is to align plugins in terms of feature (all access plugins should support both hosted and self-hosted modes).

WDYT of:
- Standalone vs embedded
- Standalone vs integration
- Self-hosted vs Teleport-Hosted

cc @r0mant